qa/tasks/cephfs/caps_helper.py

```diff
@@ -193,3 +193,37 @@ def conduct_neg_test_for_write_caps(self, sudo_write=False):
             cmdargs.pop(-1)
             log.info('absence of write perm was tested successfully: '
                      f'failed to be write data to file {path}.')
+
+    def _conduct_neg_test_for_root_squash_caps(self, _cmdargs, sudo_write=False):
+        possible_errmsgs = ('permission denied', 'operation not permitted')
+        cmdargs = ['sudo'] if sudo_write else ['']
+        cmdargs += _cmdargs
+
+        for mount, path, data in self.test_set:
+            log.info(f'test absence of {_cmdargs[0]} perm: expect failure {path}.')
+
+            # open the file and hold it. The MDS will issue CEPH_CAP_EXCL_*
+            # to mount
+            proc = mount.open_background(path)
```

Comment:
Shouldn't we just pass the filename here? https://github.com/ceph/ceph/blob/4ede08e33632b8dbcff0e64115d5ea5232b4a41c/qa/tasks/cephfs/mount.py#L893

Reply:
For this it's okay to pass the file full path instead of filename here. For the current use cases they just passing the filename and then they will create the files in the mountpoint's root dir directly. Here we will use our own test dir.

Reply:
got it.